localstack/packages/debugpy.py

```diff
@@ -0,0 +1,46 @@
+import os
+from typing import List
+
+from localstack.packages import InstallTarget, Package, PackageInstaller
+
+# debugpy module
+DEBUGPY_MODULE = "debugpy"
+
+
+class DebugPyPackage(Package):
+    def __init__(self):
+        super().__init__("DebugPy", "latest")
+
+    def get_versions(self) -> List[str]:
+        return ["latest"]
+
+    def _get_installer(self, version: str) -> PackageInstaller:
+        return DebugPyPackageInstaller("debugpy", version)
+
+
+class DebugPyPackageInstaller(PackageInstaller):
+    def _get_install_dir(self, target: InstallTarget) -> str:
+        import inspect
+        from pathlib import Path
+
+        import localstack
+
+        # get the "root" LocalStack directory. We go two levels up from /localstack/localstack/__init__.py
+        ls_path = Path(inspect.getfile(localstack)).parent.parent
+        # TODO: make this python version independent
+        lib_path = os.path.join(ls_path, ".venv/lib/python3.10/site-packages/debugpy")
+        return lib_path
```

Comment:
As you mentioned, this does seem a bit clunky. But I think this also isn't even necessary.
This is a specific installer because it just calls pip, so this would be a case for the upcoming Python installer.
However, for now we could just:
- Add a TODO that this should be migrated to the upcoming Python installer when it's there.
- Remove this function here.
- Overwrite the `is_installed()` and just try to install the module. Something like the old installer:
  ```
  def is_installed(self) -> bool:
     try:
        import debugpy

        assert debugpy
        return True
     except ModuleNotFoundError:
        return False
  ```

Reply:
Yes I was considering something like this, but it didn't adhere to our new architecture very much. But for the `is_installed` on its own its probably ok

Reply:
Combine this with the fact that this is after all the install dir, I discarded the previous solution. But a TODO + what you suggested should work :+1: